### PR TITLE
Update start_kkp documentation with Bitbucket

### PR DESCRIPTION
**Pull request by koksay**
Update documentation for start.kubermatic for Bitbucket. 

#1009 

Signed-off-by: Koray Oksay <koray@kubermatic.com>

### CODE REVIEW

File: content/kubermatic/master/installation/start_kkp/guides/setup_repository/_index.en.md [hunk 1]
```diff
@@ -150,9 +167,9 @@ In addition to above, configure following secrets for selected cloud provider.
 | `ARM_CLIENT_SECRET`     | The Azure Client Secret for client authentication                    |
 {{% /tab %}}
 {{% tab name="GCP" %}}
-| Secret Name             | Description                                                                                |
-| ------------------------| -------------------------------------------------------------------------------------------|
-| `GOOGLE_CREDENTIALS`    | The service account key (content of `${SERVICE_ACCOUNT_NAME}-sa-key.json`) from above step |
+| Secret Name             | Description                                                                                                                                                                                        |
+| ------------------------| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `GOOGLE_CREDENTIALS`    | The service account key (content of `${SERVICE_ACCOUNT_NAME}-sa-key.json`) from above step. For **Bitbucket**, base64 encoded value is required (`base64 -w0 ${SERVICE_ACCOUNT_NAME}-sa-key.json`) |
```
Comment (michalvanco) [inline content/kubermatic/master/installation/start_kkp/guides/setup_repository/_index.en.md:172]: didn't you have another variable used for this purpose on Bitbucket? and `GOOGLE_CREDENTIALS` is then set from that env.variable?

Comment (koksay) [reply content/kubermatic/master/installation/start_kkp/guides/setup_repository/_index.en.md:172]: Do you think that's ok?

image

**Review by michalvanco — APPROVED** [review]
/approve /lgtm

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**